src/libraries/System.Globalization/tests/System/Globalization/TextInfoTests.cs

```diff
@@ -394,6 +394,7 @@ public static IEnumerable<object[]> ToUpper_TestData()
                 // we also don't preform.
                 // es-zed does not case to SS when uppercased.
                 yield return new object[] { cultureName, "\u00DF", "\u00DF" };
+                yield return new object[] { cultureName, "stra\u00DFe", "STRA\u00DFE" };
```

Comment:
What is the browser behavior for `toUpperCase()` ?

Reply:
![image](https://github.com/dotnet/runtime/assets/32700855/93551ab1-16ce-4921-a352-e938de3356ff)

Reply:
That looks correct to me :)

Reply:
It is. But here we're keeping the ICU's behavior, so no codepoint expansion.